packages/node/src/ClientAuthentication.ts

```diff
@@ -74,6 +74,7 @@ export default class ClientAuthentication extends ClientAuthenticationBase {
 
     if (loginReturn !== undefined) {
       this.fetch = loginReturn.fetch;
+      this.boundLogout = loginReturn.getLogoutUrl;
```

Comment:
The logout needs to be bound on a successful login, because that's the context in which the tokens are available (because of our scope-based security approach).